trainingMaquetado/Maquetado.css

```diff
@@ -0,0 +1,49 @@
+hr {
+	background-color: #F5C85F;
+	width: 4px;
+	height: 300px;
+	margin: 60px;
+	border: none;
+  border-radius: 60px;
+  flex-shrink: 0; 
+  background: radial-gradient(#F5C85F 40%, white 60%);
+}
+
+.headline {
+	display: flex;
+	background-color: #F5C85F;
+	height: 4px;
+}
+
+
+.upper-container {	
+	display: flex;	
+	align-items: center;
+	justify-content: center;
+	height: 374px;
+}
+
+.middle-container {
+  display: flex;
+	background-color: #EEE;
+	height: 100px;
+
+}
+
+.lower-container {
+	display: flex;
+	background-color: #F5C85F;
+	height: 100px;
+}
+
+
+@media (max-width: 1140px) {
+  .esthetic {
```

Comment:
I would change this to `.phone` since it's easier to recognize it when styling it

Reply:
ps: the right way to spell it is `aesthetic`